splitio/models/telemetry.py

```diff
@@ -519,6 +520,14 @@ def record_events_value(self, resource, value):
             else:
                 return
 
+    def record_update_from_sse(self):
```

Comment:
you must to allow expansion for this telemetry because if in a future we starting to get `SPLIT_KILL` OR `SEGMENT_UPDATES` notifications with the definition inside we will want to track this telemetry too with the same method/storage.

Reply:
yes, will do